apache/trafficcontrol
# TP: adds ability to view all ds's that employ a topology / all cg's used by a topology

## What does this PR (Pull Request) do?


## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Navigate to the list of topologies - https://tp.domain.com/#!/topologies
- Select a topology to view the topology (if there are none, create one)
- Select `View Delivery Services` to see which ds's employ a given topology (if there are no delivery services listed, find a delivery service, edit it and assign it to this topology)
- Select `View Cache Groups` to see which cg's are used by a given topology (all topologies must have a minimum of one cache group)

- Documentation for topologies will be provided once the feature is fully complete
- Topology validation occurs for the most part at the API level and tests have / are being written at that level.

## The following criteria are ALL met by this PR
